bindings/java/src/main/native/Makefile.am

```diff
@@ -81,11 +81,14 @@ package : $(jarfile)
 
 # Maven install phase
 jar_DATA = $(jarfile)
-
 # Remove all compiled Java files
 clean-local:
 	-rm -rf $(java_build_dir)
 
+# Publish JUCX jar to maven central
+publish:
+	$(MVNCMD) deploy -DskipTests ${ARGS}
```

Comment:
add 2 targets:
1. "publish-snapshot" - will add -SNAPSHOT to version, check that we are on master branch
2. "publish-release" will NOT add -SNAPSHOT, check we are on branch `v*.*.x`